README.md

```diff
@@ -81,6 +81,7 @@ Deploying Haskell code onto [AWS Lambda] using [Serverless].
 ### Notes
 
 * `sls deploy function` is not supported.
+* `sls invoke local` is not supported.
```

Comment:
Why not link to a relevant issue for when they will be supported, or briefly explain why they aren't and won't be supported?